src/psicon.cpp

```diff
@@ -954,28 +953,6 @@ void PsiCon::changeProfile()
     doQuit(QuitProfile);
 }
 
-void PsiCon::doManageAccounts()
-{
-    if (!PsiOptions::instance()->getOption("options.ui.account.single").toBool()) {
```

Comment:
probably for single acount setup we could render acount modify diaog right into options. 
and if there is no any account yet, then just a label with prompt and button to create one.

Reply:
Maybe. But account modify dialog has its own dialog buttons and we probably need to hide this buttons somehow. And also send to modify dialog signals from dialog buttons of options dialog.

Reply:
to continue a chat discussion..
Well I don't mind to remove this option at all. But I can imagine a business  case when Psi set up the way making it difficult to modify options (e.g. for a company's employees), and this one seems to be useful then.

@jkarneges  do you remember what this option for?

Reply:
@Ri0n I think that's exactly the reason. Most business versions/forks of Psi have been single account.

Reply:
Single account option support implemented with disabling AccountManageDlg elements. Screenshots added below.